datepicker/views/index.html

```diff
@@ -0,0 +1,110 @@
+<index: attributes="class inline">
+{{if show}}
```

Comment:
I expect a tab as indentation here for the if statement (and thus one additional tab as indentation for below rows of course).